cmd/liftCoordinates/liftCoordinates_test.go

```diff
@@ -23,10 +23,10 @@ var LiftTests = []struct {
 
 func TestLift(t *testing.T) {
 	for _, v := range LiftTests {
-		liftCoordinates(v.chainFile, v.inputFile, "tmp.vcf", v.faFile, "tmp.unmapped", 0.95, 0)
+		liftCoordinates(v.chainFile, v.inputFile, "tmp.vcf", v.faFile, "tmp.unmapped", 0.95, false, 0)
```

Comment:
If there could be a test case for swapAlleleAB in the future, that would be great!

Reply:
I think it'd be good to add test coverage for this option in this PR before merging it in. An example, where we swap A and B.